ansible/configs/satellite-vm/default_vars.yml

```diff
@@ -0,0 +1,61 @@
+---
+
+env_type: satellite-vm 
+output_dir: /tmp/workdir                # Writable working scratch directory
+email: "{{env_type}}@example.com"
+guid: defaultguid
+
+# for file based repo
+repo_method: file 
+use_own_repos: true
+repo_version: "6.4"
+## For RHN login 
+# repo_method: rhn
+# rhsm_pool_ids:
+#   - 8a85f99b6b498682016b521dfe463949
+# rhel_subscription_user:
+# rhel_subscription_pass: 
+######
+
+deploy_local_ssh_config_location: "{{output_dir}}/"
+key_name: ocpkey                        # Keyname must exist in AWS
+use_own_key: true
+env_authorized_key: "{{guid}}key"
+set_env_authorized_key: true
+default_key_name: ~/.ssh/{{key_name}}.pem
+
+install_bastion: true
+install_common: true
+install_ipa_client: false
+tower_run: false
+update_packages: false
+update_all_packages: false
+install_satellite: True
+configure_satellite: false
+
+project_tag: "{{ env_type }}-{{ guid }}"
+
+rhel_repos:
+  - rhel-7-server-rpms 
+  - rhel-server-rhscl-7-rpms 
+  - rhel-7-server-satellite-6.4-rpms 
+  - rhel-7-server-satellite-maintenance-6-rpms 
+  - rhel-7-server-ansible-2.6-rpms
+  - rhel-7-server-extras-rpms
+
+common_packages:
+  - python
+  - unzip
+  - bash-completion
+  - tmux
+  - wget
+  - git
+  - vim-enhanced
+  - at
+  - python27-python-pip
```

Comment:
Any chance you could look at moving to python3 with these changes to be more future-proof? Several of the common roles have already been adapted to be compatible with either and we are exclusively python3 with the latest OCP on OSP config.

Reply:
Few of our modules which we use daily does not like python3 i.e. os_auth is still using python2.7 based openstacksdk so we like to keep as it is.

Reply:
Your call...we use python3 for ocp4-disco environment and that includes openstacksdk as well.